File: Python/Jenny_the_youngest_Detective.py
def missing(nums, s):
    list_str = "".join(s.split(" "))
    new_nums = sorted(nums)
    count = 0
    result = ""

    while count < len(new_nums):
        try:
            result += list_str[new_nums[count]]
            count += 1
        except:
            return "No mission today"
        
    return result.lower()


print(missing([5, 0, 3], "I love you"))

File: JavaScript/JennyTheYoungestDetective.js
function missingWord(nums, str) {
  const arrStr = str.split(" ").join("");
  const newNums = nums.sort((a, b) => a - b);
  let count = 0;
  let result = "";

  while (count < newNums.length) {
    if (arrStr[newNums[count]]) {
      result += arrStr[newNums[count]];
      count += 1;
    } else {
      return "No mission today"
    }
  } 

  return result.toLowerCase()
}

// console.assert(missingWord([5, 0, 3], "I love you") === "ivy", "erro no teste 1")
console.assert(missingWord([12, 4, 6], "Good Morning") === "No mission today", "erro no teste 2")